### DISCUSSION TITLE
configuration help

### DISCUSSION THREAD

**Discussion by nath1as** [nvim-neorg/neorg#91] [Q&A] [answered]
When I open a .norg file I get an error:
`packer.nvim: Error running config for neorg: ...ua/neorg/modules/core/integrations/nvim-compe/module.lua:22: nvim-compe not found, aborting...`

I'm using packer, latest neovim updated neorg and the setup code from README

**Comment by vhyrro** [comment, marked as answer]
Hey, we just merged the `unstable` branch into `main`. Could you try updating please? You should see things work now. The completion module is now disabled by default. You can take a look at the readme if you'd like to set it up - we have support for both nvim-compe and nvim-cmp :)

**Comment by nath1as** [comment]
a new error :)
 
`Error detected while processing BufEnter Autocommands for "*.norg":
E5108: Error executing lua /usr/share/nvim/runtime/lua/vim/treesitter/langu
age.lua:25: no parser for 'norg' language, see :help treesitter-parsers
`

**Reply by mflova** [reply]
Could you solved this issue? I am actually facing the same one. This time using bundle in my case.

**Reply by danymat** [reply]
@mflova check out: https://github.com/nvim-neorg/neorg/issues/74#issuecomment-906627223
